runtime/compiler/env/j9methodServer.cpp

```diff
@@ -1891,6 +1891,12 @@ TR_ResolvedJ9JITServerMethod::cacheResolvedMethodsCallees(int32_t ttlForUnresolv
       }
    }
 
+void
+TR_ResolvedJ9JITServerMethod::cacheFields()
```

Comment:
Yeah, I implemented in the front-end because methods `cacheField` and `getCachedField` are defined there. 
I suppose I could make these methods public and call them from the resolved method.